ncm-authconfig/src/main/pan/components/authconfig/sssd.pan

```diff
@@ -108,6 +108,7 @@ type authconfig_sssd_domain  = {
     "dns_resolver_timeout" : long = 5
     "dns_discovery_domain" ? string
     "override_gid" ? long
+    "override_shell" ? string
```

Comment:
`absolute_file_path` instead of `string`?

Reply:
Shouldn't this also be under sssd_nss?

Reply:
> override_shell (string)
    Override the login shell for all users. This option can be specified globally in the [nss] section or per-domain.
    Default: not set (SSSD will use the value retrieved from LDAP)